cachetory/serializers/json.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+import json
+from typing import Generic
+
+from cachetory.interfaces.serializers import Serializer, ValueT
+
+
+class JsonSerializer(Serializer[ValueT, bytes], Generic[ValueT]):
+
+    @classmethod
+    def from_url(cls, url: str) -> JsonSerializer[ValueT]:
+        return JsonSerializer[ValueT]()
+
+    def serialize(self, value: ValueT) -> ValueT:
+        return json.dumps(value)
+
+    def deserialize(self, data: ValueT) -> ValueT:
```

Comment:
And here `data` would the serialized data type (`str`)

Reply:
updated